### PR TITLE
Create Maps API in footer section of webpage

### PR DESCRIPTION
**Pull request by RCAVelez**
**What's new:**
Maps API is now included in the footer section with coordinates to Googleplex, no markers for now
Maps Key has been added to html 
Currently credentials key works for relez-step-2020 GCP ID, potential transition to relez-step GCP ID soon.
CSS for map of 300 pixels squared
On map div load create the map function added to JS

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/ImageUploadFormHandlerServlet.java [hunk 1]
```diff
@@ -64,7 +64,7 @@ private String getUploadedFileUrl(HttpServletRequest request, String formInputEl
     List<BlobKey> blobKeys = blobs.get("image");
 
     if (blobKeys == null || blobKeys.isEmpty()) {
-      System.err.println("Key/s are not present");
+      System.err.println("Please upload an image");
```
Comment (mecolmg) [inline portfolio/src/main/java/com/google/sps/servlets/ImageUploadFormHandlerServlet.java]: I guess I missed this change on the previous review. I'd prefer you to `throw` an error rather than print one. That means no returning empty strings. 

This possibly should be in a different PR.

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -5,6 +5,7 @@
   margin-right: auto;
   max-width: 1000px;
 }
+
```
Comment (mecolmg) [inline portfolio/src/main/webapp/style.css]: I left comments on this in a previous review. Please fix in that PR instead of here.

**Review by mecolmg — APPROVED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/ImageUploadFormHandlerServlet.java (hunk 1 above)
Comment (RCAVelez) [reply portfolio/src/main/java/com/google/sps/servlets/ImageUploadFormHandlerServlet.java]: ah ok. I'll create those throws then!

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (RCAVelez) [reply portfolio/src/main/webapp/style.css]: Yes. It was addressed already
